.circleci/config/commands/go_test.yml

```diff
@@ -116,15 +116,31 @@ steps:
           # reasons unclear.
           export DOCKER_API_VERSION=1.39
 
-          export TEST_DOCKER_NETWORK_ID=$(docker network list -q -f 'name=vaulttest')
+          export TEST_DOCKER_NETWORK_ID=$(docker network list --quiet --no-trunc --filter='name=vaulttest')
           if [ -z $TEST_DOCKER_NETWORK_ID ]; then
             TEST_DOCKER_NETWORK_ID=$(docker network create vaulttest)
+
+            # Multiple concurrent calls to 'docker network create' can succeed and cause "ambiguous network" errors from docker,
```

Comment:
Why not use a random network name?

Reply:
I guess I didn't understand why we use a non-default network in the first place, so didn't feel confident changing it. Do we actually need a custom network at all? I don't see any obvious requirement for it - AFAICT we don't attach additional containers to the same network elsewhere in the tests.

Reply:
I've tried using the Circle job number and node index to generate a unique network name that's also got some structure/predictability to it. I kept the check for whether it's already created because I think rebuilds may end up reusing the same network, but I hope it should be impossible to get two concurrent create calls with the same name with that scheme.